WMIAdventure/frontend/src/js/pages/creative/CardsCreator/atoms/SendCardPopup/SendCardPopup.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import CenteringContainer from "./styled-components/CenteringContainer";
+import TransparentBack from "./styled-components/TransparentBack";
+import PopupContainer from "./styled-components/PopupContainer";
+import ContentContainer from "./styled-components/ContentContainer";
+import InfoText from "./styled-components/InfoText";
+import Header from "./styled-components/Header";
+import DivTextArea from "./styled-components/DivTextArea";
+import TextAreaLabel from "./styled-components/TextAreaLabel";
+import TextArea from "./styled-components/TextArea";
+import DivButtons from "./styled-components/DivButtons";
+import Button from "./styled-components/Button";
+
+class SendCardPopup extends React.Component {
+    render() {
+        return (
+            <>
+                <TransparentBack show={this.props.show} onClick={this.props.hideSendCardPopupHandler}>
+                </TransparentBack>
+
+                <CenteringContainer show={this.props.show}>
+                    <PopupContainer>
+                        <ContentContainer>
+                            <Header isMobile={this.props.isMobile}>Wyślij</Header>
+                            <InfoText>
+                                Możesz dołączyć dodatkową wiadomość dla administratorów weryfikujących kartę.
+                            </InfoText>
+                            <DivTextArea>
+                                <TextAreaLabel>Wiadomość</TextAreaLabel>
```

Comment:
Label powinien posiadać atrybut htmlFor=id_takie_jak_input, aby był bezpośrednio związany z tym inputem, to też sprawi że jak klikniemy w ten label to aktywuje się przypisany do niego input